### PR TITLE
Enhanced compute.cpp file

### PR DESCRIPTION
**Pull request by RBoshae**
## **Added Fuel_Sensor_Constants.h file** 

- The file contains all the constants that were previously declared in compute.cpp.

## **Changes to compute.cpp**
- Added #include to reference Fuel_Sensor_Constants.h
- Replaced if conditionals with if else conditionals.
- Restyled the document to match Google style guides.
- Modified compute_cp function to return boolean value.

### CODE REVIEW

File: src/compute.cpp [hunk 1]
```diff
@@ -1,876 +1,420 @@
...
-const double highAssociated1CO2 = 0;
-const double highAssociated1Butane = 0.002;
-
-const double highAssociated2Methane = 0.875;
-const double highAssociated2Ethane = 0.045;
-const double highAssociated2Propane = 0.044;
-const double highAssociated2CO2 = 0;
-const double highAssociated2Butane = 0.012;
 
 void calculate_cp(int tempCnt, int pressureCnt, double &CPMethane, double &CPPropane, double &CPCO2, double &CPEthane, double &CPButane, double &thermConMethane, double &thermConEthane, double &thermConPropane, double &thermConCO2, double &thermConButane) {
```
Comment (lepatrick714) [inline src/compute.cpp]: Let's change this to a boolean function. It'll be nicer to check for it in the future instead of looking through the outputs.

**Review by lepatrick714 — COMMENTED** [review]
Looks good @RBoshae. Just one minor fix and we can ship it. :)

File: src/compute.cpp (hunk 1 above)
Comment (RBoshae) [reply src/compute.cpp]: Oh this is the boolean you were referring to. Np. Changes implemented.

**Review by lepatrick714 — APPROVED** [review]
Approved!!! Thank you @RBoshae

**Pull request merged**